code/__DEFINES/layers.dm

```diff
@@ -88,6 +88,8 @@
 #define LIGHTING_PLANE 15
 #define LIGHTING_LAYER 15
 
+#define RAD_TEXT_LAYER 15.1
```

Comment:
Shouldn't it just use the hud layer

Reply:
The creator decided to use a custom layer, the normal hud layer works just fine want me to make it use the hud layer?

Reply:
Yeah